frontend/config.ml

```diff
@@ -62,7 +62,7 @@ let pkgname : string = "islaris"
 let prefix : Filename.dirpath =
   (* Obtain a normalised path to the executable. *)
   let exe =
-    try Filename.realpath "/proc/self/exe" with Invalid_argument(_) ->
+    try Sys.executable_name  with Invalid_argument(_) ->
```

Comment:
Can you please try if the following version works as well on MacOS? 
```suggestion
    try Filename.realpath Sys.executable_name  with Invalid_argument(_) ->
```

Reply:
Yes, this change works well.